### ISSUE TITLE
Using fetch API in CF pages function works in wrangler2 but not in production

### ISSUE THREAD

**Issue by AndrewLane** [cloudflare/wrangler2#192]
I'm assuming this is just a silly mistake of some kind, but I don't know of any way to check logs or debug a "live" function that's running for a Cloudflare Pages site, and it works locally.

*What works*

1. Clone https://github.com/AndrewLane/CloudflarePagesWithWorkersAndFetchApi
2. Run `npx wrangler@beta pages dev .`
3. Exercise the functions by clicking on the two buttons on the dummy page

*What doesn't work*

1. Deploy the code from this repo to a Cloudfare Pages site
2. Exercise the functions by clicking on the two buttons on the dummy page (the one that invokes the fetch API fails and the "hello world" one succeeds)

Example: https://cloudflarepageswithworkersandfetchapi.pages.dev/

**Comment by AndrewLane** [comment]
Hmm, it looks like

```
cache: "no-cache",
```

is the problem.

**Comment by AndrewLane** [comment]
Yep, the `cache` attribute seems to be problematic, but only in Cloudflare Pages (and not wrangler2):

Works:
```
export const onRequestGet = async ({}) => {
  const url = "https://swapi.dev/api/planets/2/";
  return fetch(url, {
    method: "GET",
  });
};
```

Fails:
```
export const onRequestGet = async ({}) => {
  const url = "https://swapi.dev/api/planets/2/";
  return fetch(url, {
    method: "GET",
    cache: "no-cache",
  });
};
```

Any idea why it would work like that?

**Comment by jrf0110** [comment]
Hi there @AndrewLane 

This is an error thrown from the Cloudflare Workers runtime. The [Request.cache](https://developer.mozilla.org/en-US/docs/Web/API/Request/cache) is not implemented in Workers.

```
Uncaught (in response) Error: The 'cache' field on 'RequestInitializerDict' is not implemented.
```

**Comment by Jolg42** [comment]
Note, I found this related issue in workerd's repo https://github.com/cloudflare/workerd/issues/698
